### PR TITLE
Fix URL encoding problems

### PR DESCRIPTION
**Pull request by olivierlesnicki**
Removed encoding of the URL for iOS.

### CODE REVIEW

**Comment by Guichaguri** [comment]
Why is this needed?

**Comment by olivierlesnicki** [comment]
I'm using Firebase storage, which produces audio URL that looks a bit like this:
`https://firebasestorage.googleapis.com/v0/b/theviemusic-c2f2f.appspot.com/o/27723940%2Faudio%2FB1YqFDL0--CHILL.mp3?alt=media&token=fd8026a3-18ba-4c95-9a43-b032cb0f71bd`
(note the presence of %2 in the URN)

The problem is when added to Track Player, these URL are encoded to:
`https://firebasestorage.googleapis.com/v0/b/theviemusic-c2f2f.appspot.com/o/27723940%252Faudio%252FB1YqFDL0--CHILL.mp3?alt=media&token=fd8026a3-18ba-4c95-9a43-b032cb0f71bd`

I think it's a much better rule not to encode URL and leave this decision to the client.

File: ios/RNTrackPlayer/RNTrackPlayer.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ class RNTrackPlayer: RCTEventEmitter, MediaWrapperDelegate {
     // MARK: - Required Methods
     
     @objc(constantsToExport)
-    override func constantsToExport() -> [String: Any] {
+    override func constantsToExport() -> [AnyHashable: Any] {
```
Comment (Guichaguri) [inline ios/RNTrackPlayer/RNTrackPlayer.swift]: Please, make one modification per commit. We already have #70 for this one.

**Comment by Guichaguri** [comment]
So this is a possible fix for #40 and #44, right?

**Comment by olivierlesnicki** [comment]
It should fix both #40 and #44

**Comment by dcvz** [comment]
I agree with statement that we should leave encoding to the client. Let's merge this 👍

**Comment by AylanBoscarino** [comment]
The problem persists tho. 
I can't play a track on iOS with Firebase Storage's URL.

**Pull request merged**